gridt/src/app/login/register/register.page.html

```diff
@@ -67,6 +67,19 @@
                 Should at least be 6 characters long.
               </ion-label>
             </ion-item>
+            
+            <ion-item>
+              <ion-grid>
+                <ion-row>
+                  <input type="checkbox" [checked]="ch1" (change)="ch1= !ch1" style="padding: 2em">
```

Comment:
Calling a variable "ch1" is not good practice, it is not clear what is meant by this.